src/itinerary/index.tsx

```diff
@@ -27,12 +30,12 @@ const Itinerary = ({
 }: ItineraryProps) => {
   return (
     <ul className={cc([className, { 'kirk-itinerary--small': small }])}>
-      {headline && (
+      {isNonEmptyString(headline) && (
```

Comment:
why not directly using !isEmpty? Since Typescript will enforce the usage of string anyway

Reply:
To cover cases with strings made of whitespace spaces (e.g."   ").
I could rename isNonEmptyString to something more explicit to make it clear e.g. isEmptyOrWhitespace ?

Reply:
Oh, right for the whitespace!

About the naming it's fine like that, thanks for the explanation :)